service/validation.go

```diff
@@ -48,7 +49,20 @@ func (v *parameterValidator) Validate(value interface{}) *ParameterWarning {
 		}
 		return v.newParameterWarning("required")
 	}
-
+	if v.parameter.Repeated {
+		// Check if the value is a slice
+		if reflect.TypeOf(value).Kind() != reflect.Slice {
```

Comment:
Shoudn't we have here check for `reflect.Array`?

Reply:
I couldn't make it work with `reflect.Array` but I guess we could add both
```go
if reflect.TypeOf(value).Kind() != reflect.Slice && reflect.TypeOf(value).Kind() != reflect.Array {
```

Reply:
Why not casting to `[]interface{}`?

Reply:
See https://github.com/mesg-foundation/core/pull/680